app/uk/gov/hmrc/bindingtariffclassification/todelete/CaseData.scala

```diff
@@ -64,13 +64,22 @@ object CaseData {
       "GB")
   }
 
+  def createNewCase(app: Application = createBTIApplication,
+                 attachments: Seq[Attachment] = Seq.empty): NewCase = {
+    NewCase(
+      application = app,
+      attachments = attachments
+    )
+  }
+
   def createCase(app: Application = createBTIApplication,
+                 r: String = RandomGenerator.randomUUID(),
```

Comment:
I think we want to use a specific format rather than UUID

Reply:
This is only used by the hello world controller, can we delete that now?

Reply:
Could you do it in this PR?